calblueprint/ogsc
# Add Profile Pop-Out

# Profile Pop-Out for Signed-In User

- Created a pop-out when hovering over the signed-in user. When clicking on "profile" it redirects to the user's own profile page
- Added editing permissions (now only admins and the user logged in can edit a user's profile and only admins can delete users)

## TODO (if applicable)

- [ ] look into how to add an arrow in tailwind to the box?

## How to Test/Use PR feature

1. In the sidebar on the left, hover over the user's name section and a box should pop up with "Profile" and "Log Out"
2. Click on "Profile" and it should lead you to the user's profile page
3. If logged in as a user type other than admin, should not be able to delete own profile & cannot edit other users' profiles

## PR Checklist

Does your branch (check if true):
[X] work when you run `yarn build`?
[X] successfully run `yarn:db-migrate up` without yielding any errors?
[X] successfully run `yarn prisma introspect` without yielding any errors?
[X] successfully run `yarn dev` and support all changes that your branch intends to perform?

## Screenshots
When hovering on "Profile"
![image](https://user-images.githubusercontent.com/65790318/108594258-c6be1280-732d-11eb-9537-cdca9ee20c78.png)

After clicking on "Profile"
![image](https://user-images.githubusercontent.com/65790318/108594280-e8b79500-732d-11eb-9eea-cc1ab292fe6f.png)


CC: @sydbui
